### PR TITLE
Create troubleshooting subsection in INSTALL file

### PR DESCRIPTION
**Pull request by nickthetait**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -668,22 +668,33 @@
      ("openssl"). The libraries will be built in the top-level directory,
      and the binary will be in the "apps" subdirectory.
 
+     Troubleshooting hints:
```
Comment (mspncp) [inline INSTALL]: How about omitting naming the section just "Troubleshooting" instead of "Troubleshooting hints"?

File: INSTALL [hunk 2]
```diff
@@ -668,22 +668,33 @@
...
+
      If the build fails, look at the output.  There may be reasons
      for the failure that aren't problems in OpenSSL itself (like
-     missing standard headers).  If you are having problems you can
-     get help by sending an email to the openssl-users email list (see
+     missing standard headers).
+
+     Your best bet to fix build or test failures is to clean the build
```
Comment (mspncp) [inline INSTALL]: IMHO "Your best bet to build or test failures" sounds a bit too general. How about rewording it to something like that?

> If the build succeeded previously, but fails after a source update, it might be helpful to clean the build tree before attempting another build build.

Comment (mspncp) [reply INSTALL]: ups, only one 'build' at the end.

**Comment by mspncp** [comment]
I would suggest to squash the five commits into a single one and give it the subject

> Create troubleshooting subsection in INSTALL file

or

> INSTALL: Create troubleshooting subsection

**Comment by nickthetait** [comment]
All done and ready for review

**Comment by t-j-h** [comment]
Title has "INSALL" and it should be "INSTALL"

**Comment by mspncp** [comment]
The title of the subject line of the commit message did not contain the typo, so I refrained from complaining about it. But you are right, the PR title should be corrected, too.

**Comment by mspncp** [comment]
Merged, thanks!
